src/Bounds.cpp

```diff
@@ -738,7 +738,8 @@ class Bounds : public IRVisitor {
                 // If the argument is unbounded on one side, then the max is unbounded.
                 max = Expr();
             }
-        } else if (op->is_intrinsic(Call::likely)) {
+        } else if (op->is_intrinsic(Call::likely) ||
+                   op->is_intrinsic(Call::likely_if_innermost)) {
```

Comment:
It's not a lot of code... but maybe this would be better as a boolean argument to likely, "innermost_only"? That way, code only needs to care about one likely intrinsic, until the point they get either removed or converted to likely.